src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/changelist/changes/EntityNameChange.java

```diff
@@ -5,26 +5,14 @@
 public class EntityNameChange extends Change {
     private static final String CHANGE_NAME = "changeOfEntityName";
 
-    @JsonProperty("corporateBodyName")
-    private String corporateBodyName;
-
-    @JsonProperty("corporateBodyName")
+    @JsonProperty("proposedCorporateBodyName")
```

Comment:
Can this property be null?

Reply:
Yes, if unchanged so needs a @JsonInclude(NON_NULL)

Reply:
good catch

Reply:
Done.